docs/additional-properties-and-properties.md

```diff
@@ -0,0 +1,29 @@
+# AdditionalPropertiesAndProperties
+
+## Category
+
+Data Plane Info
```

Comment:
Seem we only have ''Error" and "Warning", what do you expect user to do with an 'Info' message ?

Reply:
In the Spectral ruleset, we use 'info' to flag a case that might not be wrong, but should be given some closer scrutiny.  Many of our Spectral rules are 'info' severity.  But I can use "Warning" as long as that does not block a PR merge.